docs/production/server.mdx

```diff
@@ -257,11 +257,15 @@ It sets up a websocket connection with a Fishjam instance and provides a simple
             case _:
```

Comment:
In line 248 there is `from jellyfish import FishjamNotifier` 💀  -> change it to `fishjam`